Libraries/Components/View/ViewPropTypes.js

```diff
@@ -22,6 +23,17 @@ const {
   AccessibilityTraits,
 } = require('ViewAccessibility');
 
+const ViewLayerType = [
+  'none',
+  'software',
+  'hardware'
+];
+
+var TVViewPropTypes = {};
+if (Platform.isTVOS) {
```

Comment:
Lint raised an error here: _'Platform' is not defined._

(Sorry for the late, late review here. This was stuck for other reasons, and I failed to get back to it in a timely manner.)

Reply:
Fixed in my rebase.